packages/composite-checkout-wpcom/src/lib/translate-cart.ts

```diff
@@ -43,6 +46,25 @@ export function translateWpcomCartToCheckoutCart( serverCart: ResponseCart ): WP
 		},
 	};
 
+	// TODO: watch out for minimal currency units while localizing this
+	const couponValueRaw = products
+		.map( product => coupon_discounts_int[ product.product_id ] )
+		.filter( Boolean )
+		.reduce( ( accum, current ) => accum + current, 0 );
```

Comment:
I don't love us having to do math on the client, but maybe there's no way around this.